src/pages/learn.js

```diff
@@ -103,6 +103,9 @@ const filterArticles = (filter, initialArticles) => {
     const filterValues = Object.keys(filter);
     return initialArticles.reduce((acc, article) => {
         for (let i = 0; i < filterValues.length; i++) {
+            if (filterValues[i] === 'page') {
```

Comment:
cool!

Reply:
Yeah I think we keep adding filters so maybe we will revisit this piece of functionality soon to clean things up for all filter types